lms/djangoapps/certificates/tests/test_tasks.py

```diff
@@ -32,4 +39,30 @@ def test_cert_task_missing_args(self, missing_arg):
 
         with patch('lms.djangoapps.certificates.tasks.User.objects.get'):
             with self.assertRaisesRegexp(KeyError, missing_arg):
-                generate_certificate(**kwargs)
+                generate_certificate.apply_async(kwargs=kwargs).get()
+
+    @patch('lms.djangoapps.certificates.tasks.generate_user_certificates')
+    @patch('lms.djangoapps.verify_student.models.SoftwareSecurePhotoVerification.user_status')
+    def test_cert_task_retry(self, user_status_mock, generate_user_certs_mock):
```

Comment:
Thanks for taking the time to change this test to verify the retry, rather than mocking the retry.  Yay!
Consider changing the name of the test to make it clear what you're doing.  Maybe `test_retry_until_verification_status_updates`.